src/pvnet_app/forecast_compiler.py

```diff
@@ -97,6 +102,18 @@ def validate_forecast(
         raise Exception(
             "FAIL: Forecast has critical fluctuations (≥500 MW up and down).")
 
+    # Validate based on sun elevation > 10 degrees
+    solpos = pvlib.solarposition.get_solarposition(
+        time=national_forecast_values.index,
+        latitude=55.3781,  # UK central latitude
+        longitude=-3.4360,  # UK central longitude
+        method='nrel_numpy'
+    )
+
+    # Check if forecast values are > 0 when sun elevation > 10 degrees
+    elevation_above_10 = solpos["elevation"] > 10
```

Comment:
could you make this `10` and environment variable that we can change. Potentially somelike FORECAST_VALIDATE_SUN_ELEVATION_LOWER_LIMIT`